book/src/proposals/booting-from-snapshot.md

```diff
@@ -0,0 +1,262 @@
+# Booting From a Snapshot
+
+## Problem
+
+A validator booting from a snapshot should confirm that they are caught up to
+the correct cluster before voting. There are two major risks if they do not:
```

Comment:
This is hard to read. It says "There are two major risks:" and then there are two sections, neither of which are risks.

Also, you mention two things the validator should do, catch up and verify they are looking at the correct cluster. Then you mention two risks if they don't. If they don't do one of those things? both of those things? Same for both risks or just two different things?